src/Types.hs

```diff
@@ -640,6 +657,7 @@ defaultSettings :: Settings
 defaultSettings = Settings
     { _durations = Durations { _elaborationPhase = 21, _votingPhase = 21 }
     , _quorums   = Quorums   { _schoolQuorumPercentage = 30, _classQuorumPercentage = 30 }
+    , _freeze    = NotFrozen
```

Comment:
I would have used `minBound` here and then control which one it is via the `data` definition, but this is more explicit, and as long as we only have one place where we name the default constructor (which we should!), it's almost no difference when it gets refactored.